tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml

```diff
@@ -336,6 +330,22 @@ spec:
         - '{policy: .policy, key: .key, "effective-time": .["effective-time"]}'
         - "$(params.HOMEDIR)/report-json.json"
 
+    - name: detailed-report
+      image: quay.io/conforma/cli:latest
+      command: ["sh", "-c"]
```

Comment:
I'd suggest keeping it as simple as possible and using just "cat", (but see question on line 338).

Reply:
Answered there.